src/main/java/me/juancarloscp52/entropy/client/Screens/EntropyIntegrationsScreen.java

```diff
@@ -194,10 +197,10 @@ protected void init() {
                 help = ButtonWidget.builder(Text.translatable("entropy.options.questionMark"), (button -> {
                 })).position((this.width / 2) + 110, 30).width(20).tooltip(Tooltip.of(
                         Text.translatable(
-                            switch(platformIntegrationValue) {
-                                case 1 -> "entropy.options.integrations.twitch.help";
-                                case 2 -> "entropy.options.integrations.discord.help";
-                                default -> "entropy.options.integrations.youtube.help"; }))).build());
+                                switch(platformIntegrationValue) {
+                                    case 1 -> "entropy.options.integrations.twitch.help";
+                                    case 2 -> "entropy.options.integrations.discord.help";
+                                    default -> "entropy.options.integrations.youtube.help"; }))).build());
```

Comment:
Nit: Probably shouldn't reformat unrelated code - at least unless we set a decided code styling.